edx_credentials_themes/templates/mitpe/credentials/programs/base.html

```diff
@@ -26,17 +27,17 @@
       <div class="wrapper-accomplishment-statement">
         <div class="accomplishment-statement">
           <p class="accomplishment-statement-lead">
-            <span class="accomplishment-statement-detail copy">{% trans "This is to certify that" %}</span>
+            <span class="accomplishment-statement-detail copy">{% trans "This is to certify that" as tmsg %}{{ tmsg | htmlescape }}</span>
             <strong class="accomplishment-recipient">{{ user_data.name }}</strong>
             <img class="accomplishment-logo" src="{% static 'mitpe/images/mit-seal.png' %}"
                  alt="Massachusetts Institute of Technology">
-            <span class="accomplishment-summary copy">{% trans "has successfully completed the" %}</span>
+            <span class="accomplishment-summary copy">{% trans "has successfully completed the" as tmsg %}{{ tmsg | htmlescape }}</span>
 
             <span class="accomplishment-course">
                 <span
                     class="accomplishment-course-name">{% block program_title %} {{ program_details.title }} {% endblock %}</span>
                 <span class="accomplishment-program-type">PROFESSIONAL CERTIFICATE PROGRAM</span>
```

Comment:
I think this should be translated. We translate it on the marketing site.

Reply:
This is also mitpe, should I still add trans anyway?

Reply:
No, I think your point about it being out of scope is correct. If you create a ticket for mitpe translation issues, maybe just add it there?